cypress/support/pages/login_management/pageLoginManagement.js

```diff
@@ -141,10 +141,7 @@ class Login_Management {
 	}
 
 	clickOnSubmitButton() {
-		cy.get(Login_Management.#loginFormSelector)
-			.should("be.visible")
-			.submit()
-			.wait(8000);
+		cy.get(Login_Management.#loginSubmitBtn).should("be.visible").click().wait(8000);
```

Comment:
same as above, it will not resolve the issue

Reply:
done!